src/tileray.h

```diff
@@ -63,7 +63,7 @@ class tileray
 
         void advance( int num = 1 ); // move to the next tile (calculate last dx, dy)
         void clear_advance(); // clear steps, leftover, last_dx, and last_dy
-        bool end();      // do we reach the end of (dx,dy) defined ray?
+        int get_steps() const;  // how many steps we advanced
```

Comment:
Since no one uses this function, do we need it?

Reply:
Not sure which one you're referring to. "end()" is unused, that's why I'm deleting it. "get_steps()" is called from a bunch of places in my code.

Reply:
Okay, I browsed for get_steps() and couldn't find it.  If you're using it, that's fine.